pkg/scheduler/schedule_one_test.go

```diff
@@ -1266,7 +1278,7 @@ func TestSchedulerBinding(t *testing.T) {
 			// Checking extenders binding.
 			for i, ext := range test.extenders {
 				wantBound := i == test.wantBinderID
-				if gotBound := ext.(*fakeExtender).gotBind; gotBound != wantBound {
+				if gotBound := ext.(*fakeExtender).gotBind; gotBound != wantBound && !ext.IsIgnorable() {
```

Comment:
If extender bind is failed, the gotBound will be false, and the wantBound is false since wantBinderID is set to -1 in your case. 
So `gotBound` will always equals to `wantBound` with or without condition `!ext.IsIgnorable()`. 
Is condition `!ext.IsIgnorable()` still necessary?

Reply:
Thanks for reminding.